homeassistant/components/glances/const.py

```diff
@@ -29,8 +29,9 @@
     "process_thread": ["Thread", "Count", "mdi:memory"],
     "process_sleeping": ["Sleeping", "Count", "mdi:memory"],
     "cpu_use_percent": ["CPU used", "%", "mdi:memory"],
-    "cpu_temp": ["CPU Temp", TEMP_CELSIUS, "mdi:thermometer"],
     "docker_active": ["Containers active", "", "mdi:docker"],
     "docker_cpu_use": ["Containers CPU used", "%", "mdi:docker"],
     "docker_memory_use": ["Containers RAM used", "MiB", "mdi:docker"],
 }
+TEMP_SENSORS = {"cpu_temp": ["CPU Temp", TEMP_CELSIUS, "mdi:thermometer"]}
+CPU_TEMP = "cpu_temp"
```

Comment:
I don't see a benefit to break it out of the existing dict in a new one.

Reply:
I broke it out because I don't want to create only one sensor for this type. I want to create multiple sensors.

Reply:
You are doing a lookup for every sensor that will be created thus it doesn't matter if you stick with `SENSOR_TYPES` or use the new `TEMP_SENSORS` dict. The lookup is performed for the same key but for different dicts.

Reply:
I have updated the code so that sensors are added dynamically based on the collected data from glances_api.
For `fs` in case of multiple detected disks all will be added.
For `sensors` also all sensors are added.
For `sensor_types` whose key doesn't have values the sensor is not added at all.